src/components/owner-quote/owner-quote.js

```diff
@@ -0,0 +1,70 @@
+// owner-quote.js
+import React from "react";
+import ownerPicture from "../../assets/images/owner-quote-callie-photo.png";
+import styled from "styled-components";
+import variables from "../../styles/styled-components.js";
+import "../../styles/fonts.css";
+
+// @media only screen and (min-width : 480px) {
+// }
+// @media only screen and (min-width : 768px) {
+// }
+// @media only screen and (min-width : 1224px) {
+// /* Styles */
+// }
+
+const OwnerCard = styled.div`
+  display: flex;
+  padding: 38px;
+`;
+
+const OwnerPhoto = styled.img`
+  display: block;
+  width: 133px;
+  height: 125px;
+
+  @media only screen and (min-width: 768px) {
+    width: 249.45px;
+    height: 266px;
+  }
+`;
```

Comment:
Try use `em` instead of `px`. It will make your height and width more proportionally scalable. 
[CSS Units](https://www.w3schools.com/cssref/css_units.asp)
[em to px converter](https://www.w3schools.com/cssref/css_pxtoemconversion.asp)